signup-service/src/main/java/io/mosip/signup/services/RegistrationService.java

```diff
@@ -142,7 +142,7 @@ public GenerateChallengeResponse generateChallenge(GenerateChallengeRequest gene
         if(cacheUtilService.isIdentifierBlocked(identifier))
             throw new SignUpException(ErrorConstants.IDENTIFIER_BLOCKED);
 
-        if(!generateChallengeRequest.isRegenerate()) {
+        if(!generateChallengeRequest.isRegeneratedChallenge()) {
```

Comment:
Naming `isRegeneratedChallenge()` is good but `isRegeneratingNewChallange()` or `isRegenerateNewChallange()`  is also another good opt IMO.